### PR TITLE
Get set property text value

### PR DESCRIPTION
**Pull request by epeach**
[Axosoft](https://codeorg.axosoft.com/viewitem?id=1319&type=features&force_use_number=true)

1. set/get properties for Text Area using value
2. set/get properties for Text Input using text
3. set/get properties for Dropdown using text

### CODE REVIEW

**Comment by epeach** [comment]
Realized this area is lightly tested. Tests incoming.

**Comment by epeach** [comment]
Finally got around to adding tests. @caleybrock Please take a look when you get a chance!

File: apps/test/unit/applab/designModeTest.js [hunk 1]
```diff
@@ -99,3 +99,56 @@ describe('makeUrlProtocolRelative', () => {
     });
   });
 });
+
+describe('setProperty and read Property', () => {
+  // Create HTML elements to get/set
```
Comment (caleybrock) [inline apps/test/unit/applab/designModeTest.js]: I've never seen the setup just inside the `describe` block. I'd put it in a `beforeEach` block, but @islemaster do you need the `beforeEach`?

Comment (epeach) [reply apps/test/unit/applab/designModeTest.js]: Discussed offline with Brad - outcome: it's cleaner to have this is a beforeEach. Added in latest commit.

**Pull request merged**